File: customized-checkout/customers/nodejs/search_customer.js
var MP = require ("mercadopago");

var mp = new MP ("ACCESS_TOKEN");

var filters = {
    "email": "your.payer@email"
}

var customer = mp.get ("/v1/customers/search", filters);

customer.then (function (customer) {
		console.log (customer);
	},
	function (error) {
		console.log (error);
	});

File: customized-checkout/customers/java/search_customer.java
import com.mercadopago.MP;
import org.codehaus.jettison.json.JSONObject;

public class MyClass {
	public static void main (String args[]) {
		MP mp = new MP ("ACCESS_TOKEN");

		Map<String, Object> filters = new HashMap<String, Object> ();
		filters.put("email", "your.payer@email");

		JSONObject customer = mp.get ("/v1/customers/search", filters);

		System.out.println(customer.toString());
	}
}
